constants.py

```diff
@@ -0,0 +1,3 @@
+urls=[]
+urls_to_check=[]
+recurse_number=0
```

Comment:
```suggestion
urls = []
urls_to_check = []
recurse_number = 0
```
Have spaces between operations